modules/@apostrophecms/module/index.js

```diff
@@ -509,14 +507,6 @@ module.exports = {
         return _.get(self.options, dotPathOrArray, def);
       },
 
-      // If `name` is `manager` and `options.browser.components.manager` is set,
-      // return that string, otherwise return `def`. This is used to decide what
-      // Vue component to instantiate on the browser side.
-
-      getComponentName(name, def) {
-        return _.get(self.options, `browser.components.${name}`, def);
```

Comment:
An alternative would be to update `getComponentName` to use `options.components` instead. No modules are currently using that option, but some, like the modal module, do open the door:

```
  getBrowserData(req) {
    return {
      modals: self.modals,
      components: {
        the: options.components.the || 'TheAposModals',
        confirm: options.components.confirm || 'AposModalConfirm'
      }
    };
  }
```

Reply:
... I'll do that.